lib/Index/Index.cpp

```diff
@@ -825,8 +826,9 @@ static bool isTestCandidate(ValueDecl *D) {
 
   // 5. ...is of at least 'internal' accessibility (unless we can use
   //    Objective-C reflection)...
-#if SWIFT_OBJC_INTEROP
-  if (D->getFormalAccess() < Accessibility::Internal)
+#if !SWIFT_OBJC_INTEROP
```

Comment:
I know this conditional was here before, but it is not correct.  Swift is a cross-compiler, so to query target properties we should be performing a runtime check on the current configured target, not a `#if`.  (Consider a SourceKit running on OS X, but editing a project targeting Linux.)